installation-linux/README.md

```diff
@@ -53,14 +53,14 @@ Running the following commands will install meeko and ringtail in
 the currently active micromamba environment.
 
 ```
-pip install meeko==0.6.0a3
```

Comment:
seems like ringtail install a previous version of meeko, so we should probably install the right version after (or update it from ringtail release).
Also, now this would still not work until the Meeko release from develop

Reply:
ringtail does not install meeko (?). It seems like the current ringtail works with meeko develop. Maybe you already talked to someone in the lab, but I can help with specific things that didn't work for you

Reply:
my bad, seems like it does install meeko 0.5.1 but when installing it with micromamba, not with pip. We can remove this change then